dicomImport/matRad_scanDicomImportFolder.m

```diff
@@ -30,7 +30,9 @@
 % LICENSE file.
 %
 % %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-
+%% print current status of the import script
+fprintf('The following things are not implemented:\n');
+fprintf('not implemented - \n use automatically the right dose file (at the moment just list all)\n');
 %% get all files in search directory
```

Comment:
Maybe in the interest of good marketing we can write something like

'Obs: All dose files for the selected patient will be imported.\n'
'Matching of plan and dose has to be preformed manually in the matlab command prompt or via the matRad GUI.\n'